data/mods/Xedra_Evolved/items/inventor/armor.json

```diff
@@ -0,0 +1,389 @@
+    "flags": [ "BELTED", "WATER_FRIENDLY" ],
+    "use_action": [ { "type": "attach_molle", "size": 8 }, { "type": "detach_molle" } ],
+    "armor": [
+      {
+        "encumbrance": 6,
+        "volume_encumber_modifier": 0.15,
+        "coverage": 80,
+        "covers": [ "torso" ],
+        "specifically_covers": [ "torso_hanging_back" ]
+      }
+    ]
+  },
+  {
+    "id": "inventor_leg_weight",
+    "type": "TOOL_ARMOR",
+    "category": "armor",
+    "name": { "str": "leg weight suspension system" },
+    "description": "Compact leg attachments, that distribute your weight onto ground nearby using some kind of force.  It makes you left a giant spherical footprints, but makes walking with heavy weight much easier.",
+    "weight": "1657 g",
+    "volume": "1500 ml",
+    "material": [ "steel", "copper" ],
+    "symbol": "[",
+    "looks_like": "legguard_hard",
+    "color": "light_red",
+    "flags": [ "STURDY", "OUTER" ],
+    "ammo": "battery",
+    "use_action": {
+      "target": "inventor_leg_weight_on",
+      "msg": "You turn on the weight suspension system.",
+      "active": true,
+      "type": "transform"
+    },
+    "pocket_data": [
+      {
+        "pocket_type": "MAGAZINE_WELL",
+        "rigid": true,
+        "flag_restriction": [ "BATTERY_MEDIUM", "BATTERY_HEAVY" ],
+        "max_contains_volume": "20 L",
+        "max_contains_weight": "20 kg",
+        "default_magazine": "medium_battery_cell"
+      }
+    ],
+    "armor": [
+      {
+        "material": [ { "type": "aluminum", "covered_by_mat": 100, "thickness": 0.6 } ],
+        "covers": [ "leg_l", "leg_r" ],
+        "specifically_covers": [ "leg_hip_l", "leg_hip_r" ],
+        "coverage": 70,
+        "encumbrance": 10
+      }
+    ],
+    "relic_data": {
+      "passive_effects": [ { "has": "WORN", "condition": "ACTIVE", "values": [ { "value": "CARRY_WEIGHT", "add": 35000 } ] } ]
+    }
+  },
+  {
+    "id": "inventor_leg_weight_on",
+    "copy-from": "inventor_leg_weight",
+    "name": { "str": "leg weight suspension system" },
+    "type": "TOOL_ARMOR",
+    "power_draw": "5 W",
+    "revert_to": "inventor_leg_weight",
+    "use_action": { "target": "inventor_leg_weight", "msg": "You turn off the weight suspension system.", "type": "transform" }
+  },
+  {
+    "id": "inventor_jump_boots",
+    "type": "TOOL_ARMOR",
+    "category": "armor",
+    "name": { "str": "jump driver" },
+    "description": "Heavily mechanized boots with small integrated battery inside.  Upon activation make you jump heavily.",
+    "weight": "1657 g",
+    "volume": "1500 ml",
+    "material": [ "steel", "copper" ],
+    "symbol": "[",
+    "looks_like": "legguard_hard",
+    "color": "light_red",
+    "flags": [ "STURDY", "OUTER" ],
+    "ammo": "battery",
+    "charges_per_use": 150,
+    "use_action": { "type": "cast_spell", "spell_id": "jump_boots_leap", "no_fail": true, "level": 0 },
+    "pocket_data": [
+      {
+        "pocket_type": "MAGAZINE_WELL",
```

Comment:
battery magazine isn't an integrated battery.  Do you want to change the pocket to an integrated battery or the description?